pkgs/by-name/cd/cd-up/package.nix

```
@@ -0,0 +1,39 @@
+{
+  fetchFromGitHub,
+  gcc,
+  gnumake,
+  lib,
+  stdenv,
+}:
+stdenv.mkDerivation {
+  pname = "up-core";
+  name = "up";
```

`name` is set to `${pname}-${version}` by default

```suggestion
  name = "up";
```